internal/entitycache/cache.go

```diff
@@ -61,12 +61,23 @@ type Cache struct {
 	baseEntities stash
 }
 
+var requiredEntityFields = map[string]int{
+	"_id":             1,
+	"promulgated-url": 1,
+	"development":     1,
+	"baseurl":         1,
+}
+
+var requiredBaseEntityFields = map[string]int{
+	"_id": 1,
```

Comment:
You would, but from the point of view of this package, I think it's better to be explicit.